views/layouts/MainLayout.jsx

```diff
@@ -7,46 +7,81 @@ import Sidebar from '../components/Sidebar'
 import api from '../api'
 import getCounts from '../fe-utils/countUtil'
 import { fetchSubjects } from '../fe-utils/fetchUtil'
-import { AuthContext, NotificationContext } from '../contexts'
+import {
+  AuthContext,
+  ConfigurationsContext,
+  NotificationContext,
+  SidebarContext,
+} from '../contexts'
 import { headerTitle } from './helpers'
 
 const MainLayout = ({ classes, theme }) => {
+  const temporary = 'temporary'
+  const persistent = 'persistent'
+  const dashboard = 'dashboard'
+  const [configurations, setConfigurations] = useContext(ConfigurationsContext)
   const [, setNotification] = useContext(NotificationContext)
+  const [openSidebar, setOpenSidebar] = useContext(SidebarContext)
+  const [showHeader, setShowHeader] = useState(false)
   const [user, setUser] = useContext(AuthContext)
   const [users, setUsers] = useState([])
+  const [persistSidebar, setPersistSidebar] = useState('persistent')
   const { pathname } = useLocation()
   const navigate = useNavigate()
-
-  const [openDrawer, setOpenDrawer] = useState(false)
   const [sideBarState, setSideBarState] = useState({
     totalDays: 0,
     totalStudies: 0,
     totalSubjects: 0,
   })
-
-  const toggleDrawer = () => setOpenDrawer(!openDrawer)
+  const handleSidebarOpen = () => setOpenSidebar(!openSidebar)
```

Comment:
this is toggling the sidebar, not only opening it. Let's change the constant or the implementation to be more accurate.

Reply:
[2c26c5e](https://github.com/AMP-SCZ/dpdash/pull/352/commits/2c26c5ef82ff63420783fb443c4cd7a14df00535)